library/LogLevel.re

```diff
@@ -0,0 +1,19 @@
+type t =
```

Comment:
Consider moving this file to Logger

Reply:
That's where it was initially, but I had a cyclic dependency `Config -> Logger -> Config` because `Config` needs the log level and `Logger` needs the config value in runtime